### PR TITLE
[Reopen]: Changed styles for landing page #10981 

### PR DESCRIPTION
**Pull request by hakankaradis**
Feature page, services screenshots

aws
do
open stack
vagrant
azure
gcp
docker
soft layer
mesos

### CODE REVIEW

**Comment by hakankaradis** [comment]
@sinan the pr is LGTM, can you please check the css files?

File: docs/blog/index.html [hunk 1]
```diff
@@ -4,15 +4,22 @@
...
-<div class="BlogHome-Header">
-  <div class="BlogHome-Header--wrapper">
-    <h1>Koding Blog</h1>
+<div class="Header">
+  <div class="Header--outer-wrapper">
+    <div class="Header--wrapper">
+      <div class="Header--content">
+        <h1>Koding Blog</h1>
+        <p>At vero eos et accusamus et iusto odio dignissimos ducimus qui blanditiis praesentium voluptatum deleniti
+          atque corrupti quos dolores et quas molestias excepturi sint.</p>
```
Comment (sinan) [inline docs/blog/index.html]: @ftasdelen ^^

File: docs/assets/img/landing/company-logos/logo_vagrant.svg [hunk 2]
```diff
@@ -1,17 +1,30 @@
...
+			h-8.3l-1.6,4.7h-2.6l6.6-19.4H914.3z M916.1,2175.2l-3.5-10.8l-3.5,10.8H916.1z M899.3,2182.1l-4.9-7h-4.8v7H887v-19.4h7.5
+			c4.2,0,5.7,1.8,5.7,4.8v2.8c0,2.3-0.9,3.9-3.1,4.5l5.1,7.3H899.3z M897.7,2167.5c0-1.9-0.8-2.6-3.2-2.6h-4.9v7.9h4.9
+			c2.4,0,3.2-0.7,3.2-2.6V2167.5z M872.1,2168.2v8.3c0,2.5,0.7,3.6,4.1,3.6c1,0,2.3-0.1,3.7-0.4v-6.6h2.4v8.4
+			c-1.9,0.6-4.2,0.8-6.2,0.8c-5,0-6.5-2.1-6.5-5.7v-8.5c0-3.9,2-5.7,6.9-5.7c1.7,0,3.6,0.2,5.5,0.6l-0.3,2.2
+			c-1.9-0.3-3.7-0.5-5.3-0.5C873.1,2164.7,872.1,2165.6,872.1,2168.2z M860.3,2162.7l6.6,19.4h-2.6l-1.6-4.7h-8.3l-1.6,4.7h-2.6
+			l6.6-19.4H860.3z M862.1,2175.2l-3.5-10.8l-3.5,10.8H862.1z M840.6,2182.1l-6.5-19.4h2.4l5.7,17.2l5.7-17.2h2.4l-6.5,19.4H840.6z"
+			/>
+	</g>
+</g>
+</svg>
```
Comment (sinan) [inline docs/assets/img/landing/company-logos/logo_vagrant.svg]: we need to optimize those svg's, see: https://jakearchibald.github.io/svgomg/

File: docs/_includes/home/svgs.html [hunk 3]
```diff
@@ -3,59 +3,59 @@
...
+            fill-rule="nonzero"></path>
```
Comment (sinan) [inline docs/_includes/home/svgs.html]: make sure that the svg's look good in home page @hakankaradis

File: docs/_css/_mixins.scss [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  @media (min-width: #{$tablet-width}) and (max-width: #{$desktop-width}) {
+    @content;
+  }
+}
+
+@mixin desktop {
+  @media (min-width: #{$desktop-width}) {
+    @content;
+  }
+}
```
Comment (sinan) [inline docs/_css/_mixins.scss]: good use of `@mixin`s @drewsetski 👍

File: docs/_css/code.scss [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+      border-style: solid;
+      border-color: #fff transparent;
+    }
+
+    .codeLineNumbers {
+      padding: 13px;
+      max-width: 32px;
+      -webkit-box-sizing: border-box;
+      -moz-box-sizing: border-box;
+      box-sizing: border-box;
```
Comment (sinan) [inline docs/_css/code.scss]: `@include border-box;`

**Review by sinan — CHANGES_REQUESTED** [review]
👍 @drewsetski @hakankaradis

File: docs/assets/img/landing/company-logos/logo_vagrant.svg (hunk 2 above)
Comment (hakankaradis) [inline docs/assets/img/landing/company-logos/logo_vagrant.svg]: I will update the those svg's

**Review by sinan — APPROVED** [review]

**Pull request merged**